### PR TITLE
TEC-8024 update checkstyle to latest

### PR DESCRIPTION
**Pull request by kmakowski-ebx**
### Description of Changes
checkstyle code formatting maven plugin version increased and some reformatting (organize imports) to the code has been made.

### Documentation

n/a

### Risks & Impacts

n/a

### Testing

n/a

## Final Checklist

Please tick once completed.

- [x] Build passes.
- [ ] Versioning considered (the version number in this PR is inline with semantic 
versioning requirements).
- [x] Change log has been updated.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -36,7 +36,7 @@
   representation of the urn. Instead of using `URNEntityType.name()`
   `URNEntityType.getEntityValue()` should be used to support UGCPosts.
 
-## 1.0.3 (Work in progress)
+## 1.0.3
```
Comment (MarcFletcher) [inline CHANGELOG.md]: We never actually had a 'non snapshot' release of 1.0.3 yet. I'd propose doing that after this PR has been merged, see 'Releases' section - https://github.com/ebx/ebx-linkedin-sdk

File: CodeStyle/checkstyle.xml [hunk 2]
```diff
@@ -45,17 +50,17 @@
...
             <property name="option" value="alone"/>
             <property name="tokens"
-                      value="CLASS_DEF, METHOD_DEF, CTOR_DEF, LITERAL_FOR, LITERAL_WHILE, STATIC_INIT, INSTANCE_INIT"/>
+                      value="LITERAL_FOR, LITERAL_WHILE, STATIC_INIT, INSTANCE_INIT"/>
         </module>
         <module name="OneStatementPerLine"/>
         <module name="EmptyLineSeparator">
             <property name="allowNoEmptyLineBetweenFields" value="true"/>
-            <property name="allowMultipleEmptyLines" value="false"/>
+            <property name="allowMultipleEmptyLines" value="true"/>
```
Comment (MarcFletcher) [inline CodeStyle/checkstyle.xml]: What was the reason for allowing this? I note it is different from allowMultipleEmptyLinesInsideClassMembers.

File: pom.xml [hunk 3]
```diff
@@ -229,20 +229,6 @@
           </execution>
         </executions>
       </plugin>
-      <plugin>
-        <groupId>org.apache.maven.plugins</groupId>
-        <artifactId>maven-javadoc-plugin</artifactId>
```
Comment (MarcFletcher) [inline pom.xml]: I believe Javadoc is required in order to publicly release.

File: CodeStyle/checkstyle.xml (hunk 2 above)
Comment (kmakowski-ebx) [reply CodeStyle/checkstyle.xml]: I changed it to true to align with the current state of the code. Looks like this rule has not been really enforced before upgrading to 8.29, so changing the value to `true` allows us to keep the code as it is (only imports are reorganized as I could not find proper setup for checkstyle.xml to keep imports as it is)

File: pom.xml (hunk 3 above)
Comment (kmakowski-ebx) [reply pom.xml]: yes, javadoc is still there. this was a duplicate entry

**Comment by MarcFletcher** [comment]
Closed as we were getting travis 'abuse' errors. Will reopen in the base repo.

**Pull request closed without merging**